app/routes/cfp._index.tsx

```diff
@@ -19,19 +19,34 @@ export const meta: V2_MetaFunction = () => {
 export default function Index() {
   return (
     <Layout>
-      <article className="prose prose-slate mx-auto max-w-3xl py-10 dark:prose-invert lg:prose-xl">
-        <h2>Call for Proposals</h2>
-        <p>Let's talk at PyCon ID 2023. We are open for talks submission.</p>
-        <Button asChild>
-          <a
-            href="https://papercall.io/pyconid2023"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Submit your proposal here
-          </a>
-        </Button>
-      </article>
+      <div className="flex h-full w-full items-center bg-primary-100 md:max-h-full ">
+        <div className="m-auto flex w-full max-w-7xl flex-col items-center justify-between md:flex-row lg:h-[550px] lg:px-3">
+          <div className="px-4">
+            <p className="mb-6 text-4xl font-bold md:text-6xl">
+              Call For Proposals
+            </p>
+            <p className="text-xl text-primary md:text-4xl lg:text-2xl">
```

Comment:
please use the same font styling with sponsorship page, because the font size doesn't look the same